### PR TITLE
New act type list

### PR DESCRIPTION
**Pull request by agentromain**
Changed the interface of the choose ac/op list

### CODE REVIEW

File: frog/imports/ui/GraphEditor/SidePanel/ListComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    const InOut = () => (inp !== '' && outp !== '' ? <br /> : <div />);
+    const outp = obj.meta.outputs === '' ? '' : 'Outputs : ' + obj.meta.outputs;
+
+    const onOver = () => this.setState({ over: true });
+    const onOut = () => this.setState({ over: false });
+
+    return (
+      <div value="titi" className="list-group-item">
+        <h5 style={{ fontWeight: 'bold' }}>
+          {' '}{obj.meta.name}{' '}
```
Comment (houshuang) [inline frog/imports/ui/GraphEditor/SidePanel/ListComponent.jsx]: Don't need the {'  '} spaces here?

Comment (agentromain) [inline frog/imports/ui/GraphEditor/SidePanel/ListComponent.jsx]: I think eslint put them here

Comment (houshuang) [inline frog/imports/ui/GraphEditor/SidePanel/ListComponent.jsx]: That would be prettier (which is triggered by eslint). I have prettier run on each save in my editor, so I see stuff like this right away.

Comment (agentromain) [inline frog/imports/ui/GraphEditor/SidePanel/ListComponent.jsx]: So you mean I should erase it manually every time I run npm start fix ? Or just leave it here ?

**Pull request merged**